packages/jest-runtime/bin/jest-runtime.js

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+/**
+ * Copyright (c) 2014, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+if (process.env.NODE_ENV == null) {
+  process.env.NODE_ENV = 'test';
```

Comment:
Originally I thought we won't need this, but I think it is a good default for preprocessors etc. that require a test environment. I think it's fine.